go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -1177,3 +1190,21 @@ func (bnc *BaseNetworkController) wasPodReleasedBeforeStartup(uid, nad string) b
 	}
 	return bnc.releasedPodsBeforeStartup[nad].Has(uid)
 }
+
+func checkBridgeMapping(physicalNetworkName string) error {
+	external_ids, stderr, err := util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
+		"external_ids:ovn-bridge-mappings")
+	if err != nil {
+		return fmt.Errorf("failed to get ovn-bridge-mappings stderr:%s (%v)", stderr, err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to get ovn-bridge-mappings from the OVS global table. stderr: %q (%v)", stderr, err)
```

Reply:
%q doesnt fit for stderr i think ?
for example if it has \n, it will print them escaped iiuc,
in any case, since it is not a name, but a string, %q shouldn't be used imho